packages/next-swc/crates/napi/src/next_api/project.rs

```diff
@@ -607,3 +612,122 @@ pub fn project_update_info_subscribe(
     });
     Ok(())
 }
+
+#[turbo_tasks::value]
+#[derive(Debug)]
+#[napi(object)]
+pub struct TracedSource {
+    pub frame: StackFrame,
+    pub source: String,
```

Comment:
What is `source`? How is it different than the frame's `file`?

Reply:
It's the source code of the asset.